Makefile

```diff
@@ -5,13 +5,22 @@
 # Variables
 BINARY_NAME = kubectl-oadp
 INSTALL_PATH ?= /usr/local/bin
+VERSION ?= $(shell git describe --tags --always --dirty 2>/dev/null || echo "dev")
+
+# Centralized platform definitions to avoid duplication
+PLATFORMS = linux/amd64 linux/arm64 darwin/amd64 darwin/arm64 windows/amd64
```

Comment:
windows/arm64 pls?

Reply:
match https://kubernetes.io/releases/download/#binaries

Reply:
oh and the IBM boss would like if you add linux/ppc64le linux/s390x

Reply:
Updated